lib/nerves_hub_link/configurator.ex

```diff
@@ -3,11 +3,12 @@ defmodule NervesHubLink.Configurator do
   alias __MODULE__.{Config, Default}
   require Logger
 
-  @device_api_version "1.0.0"
+  @device_api_version "2.0.0"
   @console_version "2.0.0"
 
   defmodule Config do
-    defstruct connect: true,
+    defstruct archive_path: "/data/archives/",
```

Comment:
I'm torn on this. I'd rather keep downloads in a consolidated location like`#{data_path}/archives` so all NHL download data is scoped, but I could see clients potentially wanting it somewhere else. 🤔 


I think I'd vote for having it in the consolidated location and not configurable. We wouldn't make any assertion on the file existing or not and instead present the info to `NervesHubLink.Client` to let them decide if it should be downloaded now, later, or acted on (like with firmware updates). What do you think?